docs/user-guide/security-features.md

```diff
@@ -1,152 +1,152 @@
+However, users have the flexibility to explicitly modify these policies as desired.
 
-**Note:-**
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/global-security-policy.jpg)
 
-Policies work in hierarchical order.
+## Configure Cluster Security Policy
 
-Order to be followed- First Global and second Cluster and so on as you can see the order of the options
 
-![](../.gitbook/assets/security-feature-global-security-policies.png)
+In Global Security Policies, there are two options: `Block` and `Allow`. Cluster Security Policies have an additional option called `Inherit`.
 
-**Some examples of how policies can be defined**
+When Inherit is selected, the policy adopts settings from higher-level options. For example, if critical severity levels are blocked globally, they will also be blocked in Cluster Security Policies. Changing the global policy to allow critical levels will also allow them in Cluster Security Policies. Explicit changes can be made to these policies.
 
-Users can block all the critical vulnerabilities and allow the moderate and low vulnerabilities or Users can block all vulnerabilities or users can block all vulnerabilities for one application and can block only Critical vulnerabilities for other applications.
+To block critical vulnerabilities globally but allow them in specific clusters:
 
-## Configure Global Policy
+1. Select the desired cluster.
+2. Change the critical setting to allow.
+3. This change only affects the policy of the selected cluster without impacting others or the global policy.
 
-To configure these policies, click on the drop-down option of the severity levels and select Block or Allow.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/cluster-security-policy.jpg)
 
-![](../.gitbook/assets/security-feature8%20%282%29.png)
+## Configure Environment Security Policy
 
-## Configure Cluster Security Policy
+Environment Security Policies, like Cluster Security Policies, offer three options: Block, Allow, and Inherit.
 
-In the Global Security Policies, there are only two options available- Block and Allow. But in other options, we have an extra option named `Inherit`.
+The Environment Security Policy inherits its settings from the Cluster Security Policy, following a hierarchical structure where each level inherits the policy from its upper level.
 
-As the name itself gives you an idea about this option, it fetches the policy of its upper-level options, if we choose to inherit in the drop-down.
+When you select an environment, it automatically adopts the policy of the associated Cluster. For example, if critical-level vulnerabilities are blocked globally but allowed in the Cluster Security Policy, the Environment Security Policy will inherit this allowance. Consequently, critical-level vulnerabilities will also be allowed in the Environment Security Policy.
```

Comment:
When you select an environment, it automatically adopts the policy of the associated cluster. For example, if critical-level vulnerabilities are blocked globally but allowed in the `Cluster Security Policy`, the `Environment Security Policy` will inherit this allowance. Consequently, critical-level vulnerabilities will also be allowed in the `Environment Security Policy`.